src/System.IO.MemoryMappedFiles/src/System.IO.MemoryMappedFiles.csproj

```diff
@@ -27,6 +27,24 @@
     <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedViewStream.cs" />
     <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedFileRights.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uapaot' ">
```

Comment:
I noticed S.Linq.Expressions has these in the project file, I suspect it means they should have a uapaot configuration as they don't seem relevant to uap.
```xml
  <PropertyGroup Condition="'$(TargetGroup)' == 'uapaot'">
    <DefineConstants>$(DefineConstants);FEATURE_DYNAMIC_DELEGATE</DefineConstants>
  </PropertyGroup>
  <ItemGroup Condition="'$(TargetGroup)' == 'uapaot'"> 
    <EmbeddedResource Include="Resources\$(AssemblyName).rd.xml" />
    <ReferenceFromRuntime Include="System.Private.CoreLib.DynamicDelegate" />
  </ItemGroup>
```